sql/item_latents.sql

```diff
@@ -23,7 +23,7 @@ CREATE TABLE IF NOT EXISTS `item_latents` (
 
 -- INSERT INTO `item_latents` VALUES(itemID, modId, modValue, latentId, latentParam);
 
-INSERT INTO `item_latents` VALUES(10737, 63, 10, 513, 64); -- Abyss Sollerets +2, Enhances "Last Resort" effect
+INSERT INTO `item_latents` VALUES(10737, 63, 10, 51, 64); -- Abyss Sollerets +2, Enhances "Last Resort" effect
```

Comment:
still wrong.
```cpp
 LATENT_JOB_LEVEL_ABOVE = 51, //PARAM: level
```
I think it was meant to be 
```cpp
LATENT_STATUS_EFFECT_ACTIVE = 13, //status effect on player - PARAM: EFFECTID
```

Reply:
That's probably my fault... I believe I steered him down that road. Oops...